### DISCUSSION TITLE
Speak Readings Focus

### DISCUSSION THREAD

**Discussion by sirlandos** [NightscoutFoundation/xDrip#2135] [General]
When I go for a run, I enable "Speak Readings".  I'm also playing music.  The readings are difficult to hear over the music.  Is there a setting that will reduce the music volume when readings are spoken?  

I've noticed that Strava (another app tracking distance) will reduce the music volume when speaking run distance/times.  I'm new to the code and have only reviewed a small portion, but I didn't see an AudioFocusRequest in SpeechUtil.java.  Is this (or something like it) implemented?  Here is a reference: https://stackoverflow.com/questions/3043595/fade-in-and-out-music-while-speaking-a-text

Oddly I never had this issue with my old Samsung S7 phone.  I recently "upgraded" to an S20 and now this feature and music conflict.

**Comment by Navid200** [comment]
If I'm not mistaken, we have recently started using audio focus for some alerts.

Would you please open an issue as a feature request for this?
I will then tag individuals who may be interested and may be able to help.

**Reply by sirlandos** [reply]
Ok
